elastic/eui
# [EuiSuperUpdateButton] use Partial for toolTipProps

## Summary

closes: #7689 

This PR changes the type from `EuiSuperUpdateButton`s prop `toopTipProps` to `Partial<EuiToolTipProps>` as the `children` property is not actually required for the use case.

## QA

- checkout the PR `gh pr checkout 7689`
- pass `toolTipProps` on `updateButtonProps` for a `EuiSuperDatePicker` EUI docs example (e.g. `src-docs/src/views/super_date_picker/super_date_picker_config.tsx`)
```ts
updateButtonProps={{
  fill: showFill,
  toolTipProps: {
    content: 'foo',
    delay: 'long',
    position: 'bottom',
  },
}}
```
- observe no typescript error for missing `children` prop in the IDE
- observe no typescript error when running `yarn test`

### General checklist

- Browser QA - N/A
- Docs site QA - N/A
- Code quality checklist - N/A
- Release checklist
